### DISCUSSION TITLE
请问能否手动提高一个词的词频的顺序问题

### DISCUSSION THREAD

**Discussion by CicadaSeventeen** [fcitx/fcitx5#886] [Q&A]
在我这边，"w"的第一候选是“为”而不是“我”。当我故意重复打多次“我”之后，第一候选能变成“我”，但是常常过些时候又变回了“为”。

类似的问题发生在“ershi”，尽管我更多地使用“而是”，但是第一候选常常是“二十”。

这类问题使得我比较容易打错字。请问有没有办法能强制提高一个词的顺序呢？

谢谢

**Comment by CornWorld** [comment]
同问

**Comment by wengxt** [comment]
首先，建议输入句子，而不是输入词，上下文的内容会给他更多的信息来提示。
然后，概率会随着你输入的内容而衰减，你最近输入的会提供更高的得分
最后，你可以用自定义词组功能来固定某些顺序，例如

![image](https://github.com/user-attachments/assets/b6e04a94-b4c1-4b9d-8ed8-4ce10f5220ff)

输入 w 的第一个就一定是 我
